client/src/components/Header/components/universal-nav.css

```diff
@@ -492,7 +492,7 @@ button.nav-link[aria-disabled='true'] {
 }
 
 .ais-SearchBox-submit:focus:not(:focus-visible) {
-  outline: none;
+  outline-color: transparent;
```

Comment:
```suggestion
```

Revert back to outline: none. This is for non-keyboard focus in which we don't want a focus outline.